easybuild/tools/filetools.py

```diff
@@ -408,7 +508,7 @@ def calc_block_checksum(path, algorithm):
         blocksize = algorithm.blocksize * 262144  # 2^18
     except AttributeError, err:
         blocksize = 16777216  # 2^24
-    _log.debug("Using blocksize %s for calculating the checksum" % blocksize)
+    _log.debug("Using blocksize %s for calculating the checksum", blocksize)
```

Comment:
I know PyLint complains about this, but I don't think this approach works, but string doesn't get formatted properly like this...